terraform/modules/hub/egress_proxy.tf

```diff
@@ -50,6 +50,7 @@ locals {
     "auth\\.docker\\.io",                                                      # Docker Hub
     "production\\.cloudflare\\.docker\\.com",                                  # Docker Hub
     "www\\.e\\.${var.deployment}\\.signin\\.service\\.gov\\.uk",               # Metadata
+    "config.e\\.${var.deployment}\\.signin\\.service\\.gov\\.uk",              # Confi
```

Comment:
Typo in the comment.

Reply:
Good catch, but I will need to remove this line anyway since the apps are expected to talk to the config internally without having to go through the egress-proxy.